app/sim_test.go

```diff
@@ -8,24 +8,27 @@ import (
 	"testing"
 
 	"github.com/CosmWasm/wasmd/x/wasm"
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
```

Comment:
I thought fixing sims and params was a different issue.

I mean, good to update the format and have it compile, but you can focus on sims after patch net is out

Reply:
Correct, simulations are not part of this PR. I was just ensuring it is still compiling and not have many changes to the default simapp example